resources/keb/templates/authorization-policy.yaml

```diff
@@ -151,6 +151,7 @@ spec:
       values:
       - {{ .Values.oidc.groups.admin }}
       - {{ .Values.oidc.groups.operator }}
+      - {{ .Values.oidc.groups.viewer }}
```

Comment:
I think having having a generic option for extending it would be better. something like a `.Values.oidc.extraGroups`, and just append the list as it is.

Reply:
after discussing with Gergely, this approach is also acceptable.